app/components/AuthSection/AuthSection.css

```diff
@@ -0,0 +1,4 @@
+.toggleButton {
+  color: var(--lego-link-color);
+  font-size: 14px;
```

Comment:
Could we please use one of the `font-size` variables? I really don't like using a bunch of different sizes across the webapp. Planning on standardizing everything in ABA-358.

Currently we only have support for `12px`, `16px` and `18px`. Would `var(--font-size-small)` be too small? I'm considering making `14px` the new "small" and then call `12px` `extra-small` / `xs`, as I don't see us really needed something as small as `12px`.

I now also realize that you simply copied the code, but oh well, my opinion stays the same.

Reply:
I've changed it to `var(--font-size-small)` and I think it works fine. Making 14px the "small" seems like a good idea though.